### PR TITLE
Add actix web 4 dependencies

### PR DESCRIPTION
**Pull request by Caleb-Hill**
Adding dependencies here to declutter future PRs

### CODE REVIEW

File: rest_api/actix_web_4/Cargo.toml [hunk 1]
```diff
@@ -23,6 +23,7 @@ description = """\
 """
 
 [dependencies]
+splinter = { path = "../../libsplinter", features = ["store","store-factory"] }
```
Comment (vaporos) [inline rest_api/actix_web_4/Cargo.toml:28]: Add a space after the ,

File: rest_api/actix_web_4/Cargo.toml [hunk 2]
```diff
@@ -23,6 +23,7 @@ description = """\
 """
 
 [dependencies]
+actix-utils = "3.0.0"
```
Comment (vaporos) [inline rest_api/actix_web_4/Cargo.toml:26]: Should be "3"

File: rest_api/actix_web_4/Cargo.toml [hunk 3]
```diff
@@ -23,6 +23,7 @@ description = """\
 """
 
 [dependencies]
+actix-web = "4.0.1"
```
Comment (vaporos) [inline rest_api/actix_web_4/Cargo.toml:27]: Should be "4"

File: rest_api/actix_web_4/Cargo.toml (hunk 1 above)
Comment (Caleb-Hill) [reply rest_api/actix_web_4/Cargo.toml]: fixed

File: rest_api/actix_web_4/Cargo.toml (hunk 2 above)
Comment (Caleb-Hill) [reply rest_api/actix_web_4/Cargo.toml]: Fixed

**Review by isabeltomb — APPROVED** [review]

**Pull request merged**